website/static/js/logTextParser.js

```diff
@@ -242,26 +327,71 @@ var LogPieces = {
     },
     // The new title of node involved
     title_new: {
+        controller: function(logObject){
+            var self = this;
+            var nodeObject = logObject.embeds.original_node;
+
+            self.returnLinkForPath = function(){
+                if(paramIsReturned(nodeObject, logObject) && nodeObject.data){
+                    if (nodeObject.data.links && nodeObject.data.attributes) {
+                        return nodeObject.data.links.html;
+                    }
+                }
+                return null;
+            };
+        },
         view: function (ctrl, logObject) {
-            return returnTextParams('title_new', 'a title', logObject);
+            var url = ctrl.returnLinkForPath();
```

Comment:
Why is this a function being called here and not just a variable with values ` null ` or `nodeObject.data.links.html` ?